### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by sklein94** [cloudogu/cas#80]
The design of login form (e.g. low contrast) should be changed to improve accessibility.
What we need:
- white background
- cloudogu icon in blue
- labels in dark blue
